libraries/c_sdk/standard/ble/src/services/mqtt_ble/iot_ble_mqtt_transport.c

```diff
@@ -894,7 +894,7 @@ static MQTTBLEStatus_t handleIncomingPublish( StreamBufferHandle_t streamBuffer,
                                               size_t length )
 {
     MQTTBLEStatus_t status = MQTTBLESuccess;
-    MQTTBLEPublishInfo_t publishInfo;
+    MQTTBLEPublishInfo_t publishInfo ={0};
```

Comment:
Please format this line correctly. To do this, you can run uncrustify 0.67 with the following config: https://github.com/aws/amazon-freertos/blob/main/tools/uncrustify.cfg

Reply:
Please also take a look at 
https://github.com/aws/amazon-freertos/blob/main/libraries/c_sdk/standard/ble/src/services/mqtt_ble/iot_ble_mqtt_transport.c#L604

MQTTBLEPublishInfo_t publishInfo ={ 0 };

Reply:
Fixed